### PR TITLE
Fix the auto-completion script generation

### PR DESCRIPTION
**Pull request by oleorhagen**
Also log errors, when encountered during generation.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -53,7 +53,7 @@ install-autocomplete-scripts: build-autocomplete-scripts
 	@if which zsh >/dev/null 2>&1 ; then \
 	echo "Installing zsh auto-complete script into ${DESTDIR}${PREFIX}/usr/local/share/zsh/site-functions/" \
 	install -d $(DESTDIR)$(PREFIX)/usr/local/share/zsh/site-functions/ && \
-	install -m 644 ./autocomplete/autocomplete.zsh $(DESTDIR)$(PREFIX)/usr/local/share/zsh/site-functions/_mender-cli \
+	install -m 644 ./autocomplete/autocomplete.zsh $(DESTDIR)$(PREFIX)/usr/local/share/zsh/site-functions/_mender-cli || true\
```
Comment (lluiscampos) [inline Makefile:56]: This should not be necessary as all these lines are inside an `if` block.

File: cmd/root.go [hunk 2]
```diff
@@ -75,8 +75,14 @@ func Execute() {
 	rootCmd.LocalFlags().Parse(os.Args)
 	b, _ := rootCmd.Flags().GetBool(argRootGenerate)
 	if b {
-		rootCmd.GenBashCompletionFile("./autocomplete/autocomplete.sh")
-		rootCmd.GenZshCompletionFile("./autocomplete/autocomplete.zsh")
+		err := rootCmd.GenBashCompletionFile("./autocomplete/autocomplete.sh")
```
Comment (lluiscampos) [inline cmd/root.go:78]: So the actual error was that I didn't have an `autocomplete` directory? We are now side-tracking the problem by adding the dir to git... :ok_hand:

File: log/log.go [hunk 3]
```diff
@@ -30,6 +30,10 @@ func Err(msg string) {
 	fmt.Fprintln(os.Stderr, msg)
 }
 
+func Errf(msg string, args ...interface{}) {
+	fmt.Fprintf(os.Stderr, msg, args...)
```
Comment (lluiscampos) [inline log/log.go:34]: Nitpick: this is changed by two different commits :mag:

**Review by lluiscampos — APPROVED** [review]
Thank you :1st_place_medal:

File: cmd/root.go (hunk 2 above)
Comment (oleorhagen) [reply cmd/root.go:78]: Yup, but, I guess, actually not ignoring the error doesn't hurt anyways

File: log/log.go (hunk 3 above)
Comment (oleorhagen) [reply log/log.go:34]: Shit, squash blunder

**Pull request merged**